erdpy/projects/project_base.py

```diff
@@ -111,3 +128,61 @@ def run_tests(self, tests_directory: Path, wildcard: str = ""):
                 print("Run test for:", test_file)
                 args = [tool, test_file]
                 myprocess.run_process(args, env=tool_env)
+
+def glob_files(folder: Path, pattern: str) -> List[Path]:
+    files = folder.rglob(pattern)
+    return [Path(folder / file).resolve() for file in files]
+
+def exclude_files(files: List[Path], to_exclude: List[Path]) -> List[Path]:
+    return list(set(files).difference(to_exclude))
+
+def rename_contracts(paths: List[Path], name: Union[str, None]) -> List[Path]:
+    if name is None:
+        return paths
+    new_paths = [compute_contract_new_path(path, name) for path in paths]
+    for old_path, new_path in zip(paths, new_paths):
+        old_path.rename(new_path)
+    return new_paths
+
+def get_contract_suffix(name: str) -> str:
+    for suffix in ["-view.wasm", ".wasm"]:
+        if name.endswith(suffix):
+            return suffix
+    return ""
+
+def remove_suffix(name: str, suffix: str) -> str:
+    if not name.endswith(suffix) or len(suffix) == 0:
+        return name
+    return name[:-len(suffix)]
+
+def compute_contract_new_path(path: Path, name_hint: str) -> Path:
```

Comment:
Nice docs / examples :+1: 

Perhaps rename the function to `adjust_wasm_filename` / `rename_wasm_with_hint`? (optionally)

Reply:
`transform_wasm_path(path, hint)`

Reply:
Renamed to `adjust_wasm_filename`